packages/dds/tree/src/test/feature-libraries/optional-field/optionalField.spec.ts

```diff
@@ -280,9 +280,11 @@ describe("optionalField", () => {
 						failingDelegate,
 						fakeIdAllocator,
 						failCrossFieldManager,
-						rebaseRevisionMetadataFromInfo(defaultRevInfosFromChanges([change1]), [
-							change1.revision,
-						]),
+						rebaseRevisionMetadataFromInfo(
+							defaultRevInfosFromChanges([change1]),
+							undefined,
```

Comment:
Why is `undefined` passed here even though the change has a revision?

Reply:
Fixed